server/routes/__test__/index.unit.test.js

```diff
@@ -6,7 +6,7 @@ describe("Test the root path  '/'  ", () => {
   test('The GET method ', (done) => {
     request(app)
       .get('/')
-      .expect('Content-Type', 'text/html; charset=UTF-8')
+      .expect('Content-Type', /(text\/html; charset=utf-8)/i)
```

Comment:
I had to use regex here instead of `toLowerCase()`. Reason being, I am not aware of a way to change the response header coming in as a result of the supertest request. Using `toLowerCase()` in this `expect()` block would only change what the test is expecting to be lowercase and not the actual response which is being tested. That make sense?

The regex here has the case-insensitive flag to test for any combination of upper and lower case in the 'content-type' response.